vyper/parser/lll_node.py

```diff
@@ -124,7 +124,8 @@ def __init__(self,
                     if isinstance(self.args[2].value, int):
                         size = self.args[2].value
                     elif isinstance(self.args[2], LLLnode) and len(self.args[2].args) > 0:
-                        size = self.args[2].args / [-1].value
+                        #size = self.args[2].args / [-1].value
+                        size = 32 # WIP FIXME
```

Comment:
I'm not entirely sure the cleanest way to fix this but we probably pass a hint to the LLLnode constructor saying what the maxlen is in order to bound the estimate.